dev/fattest.simplicity/src/componenttest/containers/TestContainerSuite.java

```diff
@@ -35,6 +35,8 @@ public class TestContainerSuite {
 
     private static boolean setupComplete = false;
 
+    public static String dockerHostname = "";
+
```

Comment:
I'd rather not expose this from `TestContainerSuite` because the hostname belongs to the ExternalDockerClient.

You should be able to call `ExternalDockerClientFilter.instance().getHostname();` anywhere in your test suite and get at this value.

Reply:
Agreed 👍